src/libs/libRocket/Include/Rocket/Core/StringBase.h

```diff
@@ -185,8 +189,18 @@ class StringBase
 	inline const T& operator[](size_type index) const;
 	inline T& operator[](size_type index);
 
+	// Offer c_str() to make the interface somewhat more consistent with std::string.
+	// It eases code migration to std::string in the future should that be needed.
+	// The name also imparts better expectations about what it does better
+	// than CString() which can be confused with things like MFC's class of the same name.
+	inline const T* c_str() const;
+
 protected:	
+	inline T* _LocalBuffer();
```

Comment:
That's not what I am observing, what about _Assign, _Insert, etc. etc. ?
I'm happy to not use _ but that's why I added it in the first place.